CMSIS/Core/Include/a-profile/cmsis_gcc_a.h

```diff
@@ -889,70 +889,6 @@ __STATIC_FORCEINLINE void __set_FPEXC(uint32_t fpexc)
 
 #include "cmsis_cp15.h"
```

Comment:
can not find file cmsis_cp15.h

Reply:
I forgot to remove the include. Thanks for the hint